sdk/python/lib/pulumi/runtime/__init__.py

```diff
@@ -57,3 +57,48 @@
     register_resource_module,
     register_resource_package,
 )
+
+__all__ = [
+    # config
+    "set_config",
+    "set_all_config",
+    "get_config",
+    "get_config_env",
+    "get_config_env_key",
+
+    # mocks
+    "Mocks",
+    "set_mocks",
+    "test",
+
+    # settings
+    "Settings",
+    "configure",
+    "is_dry_run",
+    "reset_options",
+
+    # stack
+    "run_in_stack",
+    "get_root_resource",
```

Comment:
This is actually defined in `settings.py`. The `__init__.py` file is wrong, it should get it from `.settings` rather than `.stack`.